### ISSUE TITLE
NDarray.transpose not working

### ISSUE THREAD

**Issue by Banyc** [SciSharp/Numpy.NET#68]
Suppose `NDarray a` has four axis. To run the statement below:

```csharp
NDarray b = a.transpose(0, 2, 3, 1);
```

... but with error:

```text
"TypeError : transpose() takes no keyword arguments"
```

However, the following statement works fine:

```csharp
NDarray b = np.transpose(a, new int[] { 0, 2, 3, 1 });
```

**Comment by henon** [comment]
please give me a short and complete python example so that I can reproduce and test the fix

**Comment by Banyc** [comment]
Numpy.NET version:

```csharp
NDarray a = np.ones(1, 2, 3, 4);
NDarray c = np.transpose(a, new int[] { 0, 2, 3, 1 });
NDarray b = a.transpose(0, 2, 3, 1);
```

python version:

```python
import numpy as np

a = np.ones((1, 2, 3, 4))
c = np.transpose(a, (0, 2, 3, 1))
b = a.transpose((0, 2, 3, 1))
```

**Comment by henon** [comment]
fixed and pushed to nuget as version 1.25
